### DISCUSSION TITLE
How can I solve this problem?

### DISCUSSION THREAD

**Discussion by 0-031** [rdkit/rdkit#4686] [Q&A]
![image](https://user-images.githubusercontent.com/94111226/141282016-5bf93bc8-badb-472f-bfe4-e33c012ae7c3.png)

**Comment by greglandrum** [comment]
The first line in your input includes an invalid smiles: that SMILES contains `0`s when you probably mean for it to include `O`s.
If you fix that you will get a valid molecule and then you should be able to generate a mol block from that.
